### PR TITLE
Updated to Symfony 4.3

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -21,6 +22,8 @@
     </testsuites>
 
     <listeners>
+        <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener" />
```
Comment (noniagriconomie) [inline phpunit.xml.dist]: What does this do? Thx

File: config/packages/security.yaml [hunk 2]
```diff
@@ -1,8 +1,9 @@
 security:
     encoders:
         # Our user class and the algorithm we'll use to encode passwords
+        # 'auto' means to let PHP choose the best possible password hasher (Argon2 or Bcrypt)
```
Comment (noniagriconomie) [inline config/packages/security.yaml]: Php or Symfony?

File: phpunit.xml.dist (hunk 1 above)
Comment (javiereguiluz) [reply phpunit.xml.dist]: Don't know exactly ... but the official recipe includes it: https://github.com/symfony/recipes/blob/3b19bd94886555df1997b441b6cf6a1fab088f3a/symfony/phpunit-bridge/4.1/phpunit.xml.dist#L31

File: config/packages/security.yaml (hunk 2 above)
Comment (javiereguiluz) [reply config/packages/security.yaml]: Updated.

File: config/packages/security.yaml [hunk 3]
```diff
@@ -1,8 +1,9 @@
 security:
     encoders:
         # Our user class and the algorithm we'll use to encode passwords
+        # 'auto' means to let Symfony choose the best possible password hasher (Argon2 or Bcrypt)
```
Comment (yceruto) [inline config/packages/security.yaml]: `argon2i` and `bcrypt` are deprecated since 4.3
The new 'auto' choice list is `sodium` or `native`

Comment (yceruto) [reply config/packages/security.yaml]: ref: https://github.com/symfony/symfony/blob/c5772ae9bc3d9a238a9d03b97afc773ce16b93ee/src/Symfony/Component/Security/Core/Encoder/EncoderFactory.php#L87-L89

Comment (stof) [reply config/packages/security.yaml]: which are themselves using bcrypt or argon2i as algorithm.

Comment (yceruto) [reply config/packages/security.yaml]: Yes, I meant the encoder names used for this `auto` option.

**Comment by javiereguiluz** [comment]
After updating to Symfony 4.3 beta2, I see this deprecation:

image

I can't find any occurrence of `for ... if` in our templates and the deprecation message doesn't tell us which template contains the error. Any clue? Thanks.

**Comment by yceruto** [comment]
I guess it comes from the `vendor/symfony/swiftmailer-bundle/Resources/views/Collector/swiftmailer.html.twig` template.

**Comment by yceruto** [comment]
Also I see another deprecation in tests: 
```
  1x: The "DAMA\DoctrineTestBundle\PHPUnit\PHPUnitListener" class extends "PHPUnit\Framework\BaseTestListener" that is deprecated Use TestListenerDefaultImplementation trait instead.
    1x in DebugClassLoader::loadClass from Symfony\Component\Debug
```

**Comment by yceruto** [comment]
> I can't find any occurrence of for ... if in our templates and the deprecation message doesn't tell us which template contains the error. Any clue? Thanks.

@javiereguiluz I sent a PR to improve it https://github.com/twigphp/Twig/pull/3031

**Comment by fabpot** [comment]
Updating SwiftmailerBundle will remove some deprecations.

**Comment by javiereguiluz** [comment]
I no longer see deprecations after updating Symfony to 4.3 RC1 and the other dependencies too. @yceruto do you still see deprecations in tests? I can't see anything. Thanks.

**Comment by yceruto** [comment]
I still see this one https://github.com/symfony/demo/pull/974#issuecomment-496197569 https://travis-ci.org/symfony/demo/jobs/538240998#L616

**Pull request closed without merging**